src/main/events/catalogue/get-random-game.ts

```diff
@@ -1,27 +1,41 @@
 import shuffle from "lodash/shuffle";
 
-import { getRandomSteam250List } from "@main/services";
+import { Steam250Game, getSteam250List } from "@main/services";
 
 import { registerEvent } from "../register-event";
 import { searchGames, searchRepacks } from "../helpers/search-games";
-import { formatName } from "@main/helpers";
+
+let gamesList: Steam250Game[] = [];
```

Comment:
Consider using an object instead:
```ts
const state = { games: [], index: 0 };
```

I think it would be clearer, and you wouldn't use variables

Reply:
done